www/js/diary/infinite_scroll_place_item.js

```diff
@@ -30,18 +29,6 @@ angular.module('emission.main.diary.infscrollplaceitem',
     console.log("Place Item Controller called");
     console.log('config is ', $scope.config);
 
-    // timebounds is used in js/survey/enketo/enketo-add-note-button.js getPrefillTimes() function
-    // this allows us to pre-fill time and date in surveys that have 'Date', 'Start_time', and 'End_time' fields
-    $scope.timeBounds = () => {
```

Comment:
so we don't need this any more? Could this be the cause of issue (1) that Sebastian found?

Reply:
Yes, I consolidated the logic into `enketo-add-note-button.js` > `getPrefillTimes()` instead of having it split across files.
I haven't been able to reproduce that issue yet, but `getPrefillTimes()` is certainly the first place I'll check.

Reply:
Resolving this for now until you can reproduce